### PR TITLE
Add WHN monitor an experiment page

### PR DESCRIPTION
**Pull request by liz-statsig**
Add WHN monitor an experiment page

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://654d63e4eb978c0f6e38d0e1--cozy-fox-0defec.netlify.app

File: docs/statsig-warehouse-native/features/monitor-an-experiment.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      - **p-value < 0.001 and more than 0.1% absolute deviation from expected group size**: Alert (red) meaning there is likely a problem with the experiment exposures and experiment results may not be trustworthy.
+    - **Crossover users** monitors the percentage of units which are exposed to more than one experiment group for a given experiment. Statsig discards crossover users from the experiment analysis, which may reduce your sample size and statistical power. The following thresholds are used for displaying crossover users alerts:
+      - **Crossover user percentage between 1% and 10%**: Warning (yellow) meaning that the presence of a number of crossover users has been detected, but the impact on the experiment is expected to be relatively small.
+      - **Crossover user percentage over 10%**: Alert (red) indicating that there are a substantial number of crossover users, which can indicate a problem with you experiment setup.
+    - **User metrics were computed**
```
Comment (craig-statsig) [inline docs/statsig-warehouse-native/features/monitor-an-experiment.md:26]: This check confirms that we were able to join exposure data to metric data and produce experiment results.

File: docs/statsig-warehouse-native/features/monitor-an-experiment.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      - **Crossover user percentage between 1% and 10%**: Warning (yellow) meaning that the presence of a number of crossover users has been detected, but the impact on the experiment is expected to be relatively small.
+      - **Crossover user percentage over 10%**: Alert (red) indicating that there are a substantial number of crossover users, which can indicate a problem with you experiment setup.
+    - **User metrics were computed** 
+
+    Some health checks are relevant only to end-to-end statsig experminets where the statsig sdk is used for assignment, while some are only relevant for analysis experiments where assignment is handled outside of statsig.
+
+    - **Checks started** (end-to-end only) verifies that config checks are occurring.  Available shortly after the experiment starts.  
+    - **Checks have valid unit type** (end-to-end only) ensures that config checks contain the unit ID type selected for this experiment (user ID by default).  Available when checks begin.
+    - **Exposures found** (analysis only) verifies that exposures for this experiment are provided.
```
Comment (craig-statsig) [inline docs/statsig-warehouse-native/features/monitor-an-experiment.md:32]: provided, and is populated after pulse is loaded.

File: docs/statsig-warehouse-native/features/monitor-an-experiment.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+    Some health checks are used in experiments where id types are stiched: for example in a sign-up flow experiment where stitching Stable ID to User ID allows for analysis before and after a user is logged in.
```
Comment (craig-statsig) [inline docs/statsig-warehouse-native/features/monitor-an-experiment.md:34]: stitched

File: docs/statsig-warehouse-native/features/monitor-an-experiment.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+    Duplicate mapped IDs happen when the same primary ID is accosicated with multiple secondary IDs or vice versa. This happens when the primary identifier gets reset, or if there are mutliple conversions from the same primary identifier. Deduplication can reduce your sample size and may impact the overall magnitude of your results.
```
Comment (craig-statsig) [inline docs/statsig-warehouse-native/features/monitor-an-experiment.md:36]: associated

File: docs/statsig-warehouse-native/features/monitor-an-experiment.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    - **Deduplication Bias Check** reports the percentage of exposures which have duplicate mapped ids acros primary and secondary IDs in each group and performs a Chi-Squared test of independence. The following thresholds are used for displaying deduplication bias alerts:
+      - **p-value between 0.001 and 0.01**: Warning (yellow) indicating possible bias.The p-value is not low enough to confidently say that there is a real imbalance in deduplication rate between groups. In this case the recommendation is to wait and check again the next day.
+      - **p-value < 0.001**: Alert (red) meaning that there is likely a problem with deduplication being applied more to one of the treatment groups and experiment results may not be trustworthy.
+
+ 
+### Guardrail Metric Alerts
+The diagnostics page also offers metric alert monitoring
+<img width="1474" alt="Screen Shot 2022-12-06 at 9 43 45 AM" src="https://user-images.githubusercontent.com/101903926/205983990-6ccbd888-acd4-47e3-96ef-8b3a513a26bf.png"/>
+
+You can view instructions for setting up a metric alert [here] (/metrics/metric-alerts)
```
Comment (craig-statsig) [inline docs/statsig-warehouse-native/features/monitor-an-experiment.md:50]: I might remove this for now

**Pull request merged**